tests/scattering1d/test_numpy_scattering1d.py

```diff
@@ -41,8 +41,8 @@ def test_Scattering1D_T(self, backend):
             buffer = io.BytesIO(f.read())
             data = np.load(buffer)
         x = data['x']
-        J = data['J']
-        Q = data['Q']
+        J = int(data['J'])
+        Q = int(data['Q'])
```

Comment:
Why??

Reply:
When testing, J and Q loaded from the test_data_1d.npz file are array(6, dtype=int64) and array(16, dtype=int64) , respectively, not integers.

Reply:
Good point. This can be modified quickly.

Reply:
> When testing, J and Q loaded from the test_data_1d.npz file are array(6, dtype=int64) and array(16, dtype=int64) , respectively, not integers. 

ok I see